docker-compose.yaml

```diff
@@ -29,6 +29,7 @@ services:
     image: camptocamp/geomapfish-test-mapserver
     user: www-data
     environment:
+      - MS_MAPFILE=/etc/mapserver/mapserver.map
```

Comment:
No need for MAPSERVER_CONFIG_FILE here ?

Reply:
No, If we specify the MS_MAPFILE the MAPSERVER_CONFIG_FILE is optional :-)